<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CItemEdit</class>
 <widget class="QWidget" name="CItemEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="pLGroup">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="pLField1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="pLEField1"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="pLField2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="pLEField2"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="pLField3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="pLEField3"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="pLField4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="pLEField4"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pLField5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="pLEField5"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="pLField6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="pLEField6"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="pLField7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="pLEField7"/>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="pLField8">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="pLEField8"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pPBApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pLEField1</tabstop>
  <tabstop>pLEField2</tabstop>
  <tabstop>pLEField3</tabstop>
  <tabstop>pLEField4</tabstop>
  <tabstop>pLEField5</tabstop>
  <tabstop>pLEField6</tabstop>
  <tabstop>pLEField7</tabstop>
  <tabstop>pLEField8</tabstop>
  <tabstop>pPBCancel</tabstop>
  <tabstop>pPBApply</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
